### PR TITLE
Opencollective

### PR DESCRIPTION
**Pull request by Aggror**
- Remove Patreon links/logo
- Add Open Collective links/logo
- Add sponsor page
- Add Open collective blog post

### CODE REVIEW

File: _posts/2020-05-08-community-meeting-3.md [hunk 1]
```diff
@@ -47,9 +47,11 @@ We actively support the Stride open-source community and financially support the
 
 
 ## Sponsoring and gold members
-We welcome our new gold sponsor David Jeske. We would also like to thank him for contributing the work on VXGI last year. He is using Stride for one of his projects.
+OUTDATED: As off November 2021, we will use [Open collective](https://opencollective.com/stride3d)
 
-Also, a friendly reminder, that if you [sponsor Stride through Github](https://github.com/sponsors/xen2), the reward is doubled by Github. You can also still sponsor through [Patreon](https://www.patreon.com/stride3d)
+~~We welcome our new gold sponsor David Jeske. We would also like to thank him for contributing the work on VXGI last year. He is using Stride for one of his projects.~~
```
Comment (xen2) [inline _posts/2020-05-08-community-meeting-3.md:52]: I wouldn't strike this specific line (only the next one about Patreon). We don't want to erase previous donors from blog posts.

**Review by xen2 — COMMENTED** [review]
Thanks for all the changes!
I noticed some changes in css width/positions.
Did you check that it works on smaller device? (for that you just need to make Chrome/Firefox narrower, and make sure top-levels menu still work fine)

File: _posts/2020-05-08-community-meeting-3.md (hunk 1 above)
Comment (Aggror) [reply _posts/2020-05-08-community-meeting-3.md]: Good one. Just added that we now have a new platform.

**Comment by Aggror** [comment]
> Thanks for all the changes! I noticed some changes in css width/positions. Did you check that it works on smaller device? (for that you just need to make Chrome/Firefox narrower, and make sure top-levels menu still work fine)

small width
I tested this with responsive layout (f12, screen size changing) and the menu's collapse properly at certain widths.
image

medium with
image
image

large width
image

**Pull request merged**